pkg/controller/template/render.go

```diff
@@ -381,20 +381,7 @@ func cloudConfigFlag(cfg RenderConfig) interface{} {
 
 func onPremPlatformShortName(cfg RenderConfig) interface{} {
```

Comment:
This probably needs to be removed completely, but before I go through and touch all the templates I figured I would make the simple change first.